dapr/clients/grpc/_response.py

```diff
@@ -191,6 +191,11 @@ def unpack(self, message: GrpcMessage) -> None:
             ValueError: message is not protocol buffer message object or message's type is not
                 matched with the response data type
         """
+
+        if self.content_type == "application/x-protobuf":
```

Comment:
handle lower case vs upper case and content type optional suffix (`application/x-protobuf; gzip`)

Reply:
The http library already correctly parses out the suffix for us, but I'll add a test to confirm this. Will also make case sensitive.